root/wrapper.html

```diff
@@ -163,7 +163,7 @@
             </div>
         </nav>
 
-        <div class="container-fluid">
+        <div class="container-fluid" id="page-<% c.action.namespace %>">
```

Comment:
We try to avoid unneeded ids so they don't conflict with pod content.

Reply:
This allows for page specific CSS without including custom stylesheets.

Reply:
Perhaps I shouldn't be using c.action.namespace to achieve this.

Reply:
classes should be preferred to ids.

`c.action.namespace` is probably the wrong thing.  I'd say anything accessing `c` is probably wrong. Not sure what the best option is though. Possibly something injected by the view class.

Reply:
I've changed the ID to be a class, however I'm unsure whether you will like the new approach.